src/dynamic_sharing_service/policy/dynamic.policy.builder.js

```diff
@@ -39,9 +46,7 @@ module.exports = class DynamicPolicyBuilder {
    * @param {string} name
    */
   withWantedAttributeByName(name) {
-    const wantedAttribute = new WantedAttributeBuilder()
-      .withName(name)
-      .build();
+    const wantedAttribute = new WantedAttributeBuilder().withName(name).build();
```

Comment:
Was this change done by the linter? It looks like we have the old line style on lines 90-92 in this file, which I prefer.

Reply:
This might have been caused by prettier which should have been following the eslint rules.  It seems that prettier is overriding some rules so I'll move it back to how it was and avoid prettier for now.